ADD_NEW_VENDOR/Template.Application/Services/VendorService.cs

```diff
@@ -24,7 +27,15 @@ public async Task<List<Vendor>> GetAllVendorsAsync()
 
         public async Task<Vendor> AddVendorAsync(Vendor vendor)
         {
-            return await _vendorRepository.AddVendorAsync(vendor);
+            try
```

Comment:
Add lld for create and list vendor apis.
Add unit tests for all methods after creating a test project